src/main/java/ch/njol/skript/effects/EffGoatHorns.java

```diff
@@ -0,0 +1,87 @@
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.SyntaxStringBuilder;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.Goat;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.jetbrains.annotations.Nullable;
+
+@Name("Goat Horns")
+@Description("Make a goat have or not have a left, right, or both horns.")
+@Examples({
+	"remove the left horn of last spawned goat",
+	"regrow {_goat}'s horns",
+	"remove both horns of all goats"
+})
+@Since("INSERT VERSION")
+public class EffGoatHorns extends Effect {
+
+	public enum GoatHorn {
+		LEFT, RIGHT, BOTH, ANY
+	}
+
+	static {
+		Skript.registerEffect(EffGoatHorns.class,
+			"remove [the] (left horn|right:right horn|both:both horns) of %livingentities%",
+			"remove %livingentities%'[s] (left horn|right:right horn|both:horns)",
+			"(regrow|replace) [the] (left horn|right:right horn|both:both horns) of %livingentities%",
+			"(regrow|replace) %livingentities%'[s] (left horn|right:right horn|both:horns)");
```

Comment:
since you can have plural goats, `regrow the left horns of {_goats::*}` should be valid